docker-compose.yaml

```
@@ -0,0 +1,74 @@
+        condition: service_healthy
+    env_file:
+      - .env
+
+  nginx:
+    build:
+      context: .docker/nginx
+    restart: unless-stopped
+    ports:
+      - "80:80"
+      - "443:443"
+    depends_on: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)